app/test_main.py

```diff
@@ -1,3 +1,49 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age,dog_age,expected", [
+    (14, 14, [0, 0]),
```

Comment:
The expected result for (14, 14) is [0, 0], which aligns with the current implementation of `convert_to_human`. However, if the logic in `convert_to_human` is adjusted to return a fraction of the first year equivalent instead of 0, this test case will need to be updated accordingly.